src/borg/compress.pyx

```diff
@@ -126,9 +126,14 @@ class LZ4(CompressorBase):
     name = 'lz4'
 
     def __init__(self, **kwargs):
-        pass
+        self.none = get_compressor('none')
```

Comment:
Hmm, thinking about a slight micro opt and also reduction of code duplication here:

Could it just be one global NONE_COMPRESSOR object, initialized only once instead of each compressor class constructor doing it every time?

Reply:
That also should then be independendly done to the auto compressor as well, which I did not touch at all and which [in its init function does the same thing for both lz4 and none compressors](https://github.com/borgbackup/borg/blob/d85c4f9e5dbabe77f93b50f7b6053b41436b352e/src/borg/compress.pyx#L364)...

Reply:
Created that separate PR #5043.